ui/page/root/trade_page.go

```diff
@@ -117,65 +117,84 @@ func (pg *TradePage) layoutMobile(gtx C) D {
 }
 
 func (pg *TradePage) pageContentLayout(gtx C) D {
-	pg.dcrdexBtn.SetEnabled(true, &gtx)
-
-	pageContent := []func(gtx C) D{
-		pg.sectionTitle(values.String(values.StrExchangeIntro)),
-		pg.layoutAddMoreRowSection(pg.exchangeBtn, values.String(values.StrExchange), pg.Theme.Icons.AddExchange.Layout16dp),
-		pg.layoutAddMoreRowSection(pg.dcrdexBtn, values.String(values.StrDcrDex), pg.Theme.Icons.DcrDex.Layout16dp),
-	}
-
-	return cryptomaterial.LinearLayout{
-		Width:     cryptomaterial.MatchParent,
-		Height:    cryptomaterial.MatchParent,
-		Direction: layout.Center,
-	}.Layout2(gtx, func(gtx C) D {
+	return components.UniformPadding(gtx, func(gtx C) D {
 		return cryptomaterial.LinearLayout{
-			Width:  gtx.Dp(values.MarginPadding550),
-			Height: cryptomaterial.MatchParent,
+			Width:     cryptomaterial.MatchParent,
+			Height:    cryptomaterial.MatchParent,
+			Direction: layout.Center,
 			Margin: layout.Inset{
 				Bottom: values.MarginPadding30,
 			},
 		}.Layout2(gtx, func(gtx C) D {
-			return pg.Theme.List(pg.scrollContainer).Layout(gtx, len(pageContent), func(gtx C, i int) D {
-				return layout.Inset{
-					Right:  values.MarginPadding48,
-					Bottom: values.MarginPadding4,
-				}.Layout(gtx, pageContent[i])
-			})
+			return cryptomaterial.LinearLayout{
+				Width:  cryptomaterial.WrapContent,
+				Height: cryptomaterial.MatchParent,
+				Margin: layout.Inset{
+					Bottom: values.MarginPadding30,
+				},
+			}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(pg.titleContent(values.String(values.StrExchangeIntro))),
+						layout.Rigid(pg.titleContent(values.String(values.StrExchangeIntroPt2))),
+						layout.Rigid(pg.boxContentLayout),
+					)
+				}),
+			)
+
 		})
 	})
 }
 
-func (pg *TradePage) sectionTitle(title string) layout.Widget {
+func (pg *TradePage) titleContent(title string) layout.Widget {
 	return func(gtx C) D {
-		return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, pg.Theme.Label(values.TextSize20, title).Layout)
+		return layout.Inset{Bottom: values.MarginPadding0}.Layout(gtx, pg.Theme.Label(values.TextSize16, title).Layout)
 	}
 }
 
-func (pg *TradePage) layoutAddMoreRowSection(clk *cryptomaterial.Clickable, buttonText string, ic func(gtx C) D) layout.Widget {
+func (pg *TradePage) boxContentLayout(gtx C) D {
+	return cryptomaterial.LinearLayout{
+		Width:       cryptomaterial.WrapContent,
+		Height:      cryptomaterial.WrapContent,
+		Orientation: layout.Vertical,
+		Direction:   layout.Center,
+		Alignment:   layout.Middle,
+		Border:      cryptomaterial.Border{Radius: cryptomaterial.Radius(8)},
+		Padding: layout.Inset{
+			Top:    values.MarginPadding80,
+			Bottom: values.MarginPadding80,
+			Left:   values.MarginPadding124,
+			Right:  values.MarginPadding124,
+		},
+		Background: values.TransparentColor(values.TransparentBlack, 0.05),
```

Comment:
```suggestion
		Background: pg.Theme.Color.Gray4,
```
Please revert to the recommendation above. The current color looks darker than it should be.